combat-trainer.lic

```diff
@@ -189,12 +191,46 @@ class LootProcess
   def dispose_body(game_state)
     return if DRRoom.dead_npcs.length < 1
     game_state['mob_died'] = true
-
-    check_skinning(DRRoom.dead_npcs.first) if @skin
-
+      if @necro
```

Comment:
is there a way to echo my variables to see if they're getting initialized?   like:

echo("ritual_type: #{ritual_type}") if $debug_mode